### PR TITLE
Refactor RouterTabs

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: src/app/components/RouterTabs/index.tsx [hunk 1]
```diff
@@ -10,28 +10,13 @@ type RouterTabsProps = {
...
-
-  return null
-}
-
 export const RouterTabs: FC<RouterTabsProps> = ({ tabs }) => {
-  const tabsPaths = tabs.flatMap(num => num.to)
-  const routeMatch = useRouteMatch(tabsPaths)
-  const currentTab = routeMatch?.pattern?.path
+  const { pathname } = useLocation()
+  const currentTab = tabs.find(tab => matchPath(tab.to, pathname))
```
Comment (csillag) [inline src/app/components/RouterTabs/index.tsx:15]: Does this take card of the reason why flattening was used before? (in `tabs.flatMap`).

Comment (lukaw3d) [reply src/app/components/RouterTabs/index.tsx:15]: Looked like a mistake to me

Comment (csillag) [reply src/app/components/RouterTabs/index.tsx:15]: Maybe, maybe not. @buberdds wrote the previous version of the code, maybe he can explain the reason?

**Review by buberdds — APPROVED** [review]

File: src/app/components/RouterTabs/index.tsx (hunk 1 above)
Comment (buberdds) [reply src/app/components/RouterTabs/index.tsx:15]: refactor looks Ok

**Pull request merged**